rasa/shared/core/domain.py

```diff
@@ -309,8 +311,93 @@ def merge_lists_of_dicts(
 
         return self.__class__.from_dict(combined)
 
+    @staticmethod
+    def merge_domain_dicts(
+        domain1: Dict, domain2: Dict, override: bool = False
+    ) -> Dict[Text, Any]:
+        """Merge this domain dict with another one, combining their attributes.
+
+        Used when multiple domain files are configured (in a single directory).
+
+        List attributes like ``intents`` and ``actions`` will be deduped
+        and merged. Single attributes will be taken from `domain1` unless
+        override is `True`, in which case they are taken from `domain2`.
+
+        This is a fix for #10137
```

Comment:
```suggestion
```

Reply:
I don't think we normally mention bug fixes in the comments / docstrings (at least I've not seen it before). Usually we just mention it in the changelog.